internal/node/node_repositories.bzl

```diff
@@ -23,35 +23,80 @@ load(":node_labels.bzl", "get_node_label")
 load("//internal/common:check_bazel_version.bzl", "check_bazel_version")
 load("//internal/npm_install:npm_install.bzl", "yarn_install")
 
+_node_versions = {
+  "v8.9.1": {
+    "mac": {
+      "repos": [
+        "https://mirror.bazel.build/nodejs.org/dist/v8.9.1/node-v8.9.1-darwin-x64.tar.gz",
+        "https://nodejs.org/dist/v8.9.1/node-v8.9.1-darwin-x64.tar.gz",
+      ],
+      "stripPrefix": "node-v8.9.1-darwin-x64",
+      "sha256": "05c992a6621d28d564b92bf3051a5dc0adf83839237c0d4653a8cdb8a1c73b94"
+    },
+    "windows": {
+      "repos": [
+        "https://mirror.bazel.build/nodejs.org/dist/v8.9.1/node-v8.9.1-win-x64.zip",
+        "http://nodejs.org/dist/v8.9.1/node-v8.9.1-win-x64.zip",
+      ],
+      "stripPrefix": "node-v8.9.1-win-x64",
+      "sha256": "db89c6e041da359561fbe7da075bb4f9881a0f7d3e98c203e83732cfb283fa4a"
+    },
+    "linux": {
+      "repos": [
+        "https://mirror.bazel.build/nodejs.org/dist/v8.9.1/node-v8.9.1-linux-x64.tar.xz",
+        "http://nodejs.org/dist/v8.9.1/node-v8.9.1-linux-x64.tar.xz",
+      ],
+      "stripPrefix": "node-v8.9.1-linux-x64",
+      "sha256": "8be82805f7c1ab3e64d4569fb9a90ded2de78dd27cadbb91bad1bf975dae1e2d"
+    }
+  },
+  "v8.9.4": {
```

Comment:
How likely that someone needs 8.9.1 and cannot upgrade to 8.9.4? I understand the intent here is to allow choosing versions, but I thought this was more for major versions. this doesn't seem like a great example, maybe we can just change 8.9.1 to 8.9.4 without introducing configurability?

Reply:
Yes, we could (and it would solve my immediate needs), but doing this paves the way for multiple version support which is what we're really after here.

That said, nodejs has made some breaking changes in minor version updates in the past, so it's probably worth considering supporting even minor versions in this way.